tests/conftest.py

```
@@ -847,18 +847,32 @@ def golden_image_data_volume_scope_module(request, admin_client, golden_images_n
+        yield None
+    else:
+        yield from data_volume(
+            request=request,
+            namespace=golden_images_namespace,
+            storage_class=request.param["storage_class"],
+            check_dv_exists=True,
+            client=admin_client,
+        )
 
 
 @pytest.fixture()
```

Ruff (0.15.15)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)